classes/02class/exercises/c02-network01/marcio-anatrielo/c02-network01/c02-network01.txt

```diff
@@ -0,0 +1 @@
+123
```

Comment:
still need you to update this PR with the right answer

Reply:
Can you please check now if it is ok?